### PR TITLE
Restrict championship type to real countries

### PR DESCRIPTION
**Pull request by viroulep**
I noticed this when investigating a big set of `select * from Continents;` sql queries.

The big set of queries is due to our way of building the `Continent::ALL_SORTED_BY_LOCALE` variable, so it happens only once per server restart, but there is still an easy fix that I included.

But the main issue here is that I believe we don't want to be able to select our "Multiple Countries (Europe)" and such when declaring a championship type.

But I'd definitely like feedback from the Board before merging this!
(@AlbertoPdRF, @cubizh, what do you think ?)

### CODE REVIEW

**Comment by jonatanklosko** [comment]
I guess we should exclude `_Multiple Continents` as well?

**Comment by viroulep** [comment]
Right!
It was weird that both `Country` and `Continent` would expose basically the same method but would share the code, so I extracted them to a concern.

**Comment by AlbertoPdRF** [comment]
LGTM!

**Comment by viroulep** [comment]
Thanks for the feedback @AlbertoPdRF, I'll wait for a review from the WST and merge it :)

File: WcaOnRails/app/models/concerns/localized_sortable.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Appends a bunch of methods to *Cachable* classes to expose
+# a collection of sorted objects based on the user's locale.
+require 'active_support/concern'
+
+module LocalizedSortable
```
Comment (jonatanklosko) [inline WcaOnRails/app/models/concerns/localized_sortable.rb]: It says nothing about the real/fictive stuff going on here, but honestly I don't have any appropriate name on mind, so I'm fine with it ^^

Comment (viroulep) [reply WcaOnRails/app/models/concerns/localized_sortable.rb]: I know, I struggled to find an "appropriate" name :(

**Pull request merged**